MeNext/Layout/SongListView.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System.Diagnostics;
+
+using Xamarin.Forms;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace MeNext
+{
+	// Song is a POD class for song information.
+	// TODO: maybe move this out of this file? Create a more legitimate version? Use the implementation
+	// from menext/music-service?
```

Comment:
We should generally be using `ISong` whenever we can, especially since we'll need to have a reference to the `ISong` object whenever we do pretty much anything with this list.